### PR TITLE
TA 96465

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-6836

### CODE REVIEW

File: advisories/a96465.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Technical Advisory 96454
+advisory: A-96465
+summary: The global NOSQLLOGIN privilege does not restrict SQL access as it should.
+toc: true
+affected_versions: v22.2.0-v22.2.4
```
Comment (rafiss) [inline advisories/a96465.md:6]: v22.2.4 contains the fix, so it's not an affected version

**Review by rafiss — CHANGES_REQUESTED** [review]
looks good overall, except for one detail!

**Review by rafiss — APPROVED** [review]
looks good from my end, thank you! i'm not sure if we need to wait until v22.2.4 is published before posting this TA. @abhinavg6 or @dikshant might have a thought.

**Comment by gemma-shay** [comment]
Yeah, we need to wait so that the build will succeed

**Review by rmloveland — APPROVED** [review]
LGTM

**Pull request merged**